modules/ROOT/pages/configure-a-test-organization.adoc

```diff
@@ -47,24 +47,24 @@ To an organization, you create the organization, create the group hierarchy, spe
 == Export and import an organization
 
 You can export an organization definition from Bonita Studio. The exported file is in XML, so can be modified. You can import an
```

Comment:
The exported file is in XML,  => The exported file is in XML format,

Reply:
Added